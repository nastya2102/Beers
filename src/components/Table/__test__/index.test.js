import React from 'react'
import { render } from '@testing-library/react'
import Table from '../index'

describe('Table', () => {
  test('should render Table component', () => {
    const { container } = render(<Table data={1} handleClickBeer={() => {}} currentSort={"none"} handleChangeSort={() => {}}/>)

    expect(container).toBeTruthy()
  })

  test('should match snapshot Table component', () => {
    const { container } = render(<Table data={1} handleClickBeer={() => {}} currentSort={"none"} handleChangeSort={() => {}}/>)

    expect(container).toMatchSnapshot()
  })
})
