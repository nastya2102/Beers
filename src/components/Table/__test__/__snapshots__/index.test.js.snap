// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table should match snapshot Table component 1`] = `
<div>
  <div
    class="sc-jSgupP hcVCpe"
  >
    <div
      class="sc-gsTCUz gaWnRm"
      height="500px"
      opacity="0.6"
    />
    <div
      class="sc-gKsewC cZdIh"
    >
      <div
        class="sc-bdfBwQ tXWSF"
      >
        Nothing Found
      </div>
    </div>
  </div>
</div>
`;
